basiclti/basiclti-oidc/src/java/org/sakaiproject/lti13/OIDCServlet.java

```diff
@@ -126,12 +126,16 @@ private void handleOIDCAuthorization(HttpServletRequest request, HttpServletResp
 		redirect_uri = StringUtils.trimToNull(redirect_uri);
 
 		String encoded_login_hint = (String) request.getParameter("login_hint");
+		encoded_login_hint = encoded_login_hint.replace(' ', '+');  // when posted, plus signs may become supplanted by spaces
 		byte[] valueDecoded = Base64.decodeBase64(encoded_login_hint);
 		String login_hint = new String(valueDecoded);
 		if (StringUtils.isEmpty(login_hint)) {
 			state = null;
 		}
 
+		log.debug("encoded_login_hint={}", encoded_login_hint);
+		log.debug("login_hint={}", login_hint);
+
```

Comment:
@hornersa you should combine these into a single log message and differentiate it from the similar debug statement.

Reply:
I assume the class and method called (and recorded in the log statement) would distinguish one pair from another.